src/blant-utils.c

```diff
@@ -139,9 +145,8 @@ void SetGlobalCanonMaps(void)
     _numCanon = _connectedCanonicals->maxElem;
     _numConnectedCanon = SetCardinality(_connectedCanonicals);
     _numOrbits = orbitListPopulate(BUF, _orbitList, _orbitCanonMapping, _orbitCanonNodeMapping, _numCanon, _k);
-    _K = (Gordinal_type*) mapCanonMap(BUF, _K, _k);
-
-    sprintf(BUF, "%s/%s/perm_map%d.bin", _BLANT_DIR, CANON_DIR, _k);
+    _K = (Gordinal_type*) mapCanonMap(BUF, _K, _k_base);
```

Comment:
Add a comment somewhere (if there isn't already) what _k_base is and what it means.

Reply:
Done